### DISCUSSION TITLE
Björgunarleiðangur til Danmerkur - 2023/7/6

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#37] [vefsida-ummæli]
# Björgunarleiðangur til Danmerkur - 2023/7/6

Matthías villist til Danmerkur þegar hann var á leiðinni til Tromsø. Trausti gerir sitt allra besta til að bjarga honum

https://www.biskupsreglan.com/verkefni/bjorgunarleidangur-danmork

**Comment by TJ478** [comment]
Þetta er svo sorglegt, vonandi er Matthías heill á húfi.
